ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -228,12 +228,6 @@ data ChainDbEnv m blk = CDB
     -- not when hashes are garbage-collected from the map.
   , cdbNextIteratorKey :: !(StrictTVar m IteratorKey)
   , cdbNextFollowerKey :: !(StrictTVar m FollowerKey)
-  , cdbCopyLock        :: !(StrictMVar m ())
```

Comment:
I think you do want to keep the fuses in this data structure. Otherwise, someone might call `launchBgTasks` more than once on the same `ChainDB`, defeating the whole purpose of the fuse, right?

And, in general, it'd be fine for a program to have two `ChainDB`s open, as long as they were separate `ChainDB`s. And so this data structure is exactly where the fuses should live.